<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UVVino</class>
 <widget class="QMainWindow" name="UVVino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UV/Vino</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startstop_button">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>735</y>
      <width>92</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Start/Stop the data collecting</string>
    </property>
    <property name="text">
     <string>Initialize</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="time_set">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>735</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set the runtime in minutes</string>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="rangegroupbox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>710</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Range:</string>
    </property>
    <widget class="QRadioButton" name="radio_AU">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>AU</string>
     </property>
     <property name="text">
      <string>0-1 AU</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_mau">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>mAU</string>
     </property>
     <property name="text">
      <string>0 - 1000 mAU</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_mv">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>uVolt</string>
     </property>
     <property name="text">
      <string>miliVolt</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_raw">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>RAW</string>
     </property>
     <property name="text">
      <string>RAW</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcd">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>0</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>730</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>/dev/ttyUSB0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>740</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Runtime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>740</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="valve_a">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>770</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="valve_b">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>770</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="valve_c">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>770</y>
      <width>55</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>770</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valve B (%):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>770</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valve A (%):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>770</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valve C (%):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>770</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>@Time:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="valvetime">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>770</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Set the runtime in minutes</string>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>770</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mixing_addbutton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>770</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mixing_grad_check">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>770</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gradient</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="valve_a_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>790</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="valve_b_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>790</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="valve_c_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>790</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
